global-script.lua

```diff
@@ -363,9 +377,18 @@ function onLoad(saved_data)
                     o.interactable = false -- sets boards to uninteractable after reload
                 end
             end
+        else
+            enableUI()
```

Comment:
not convinced that the UI should be enabled before the game starts to what end would someone need this menu around before the game starts?

Reply:
For toggling the multihanded options before choosing spirits and for turning off the player seat controls before the game starts if desired.  (Think a more organized setting where someone might be demoing the game with preset spirits.)

Reply:
Hmm that should maybe be hooked up into the advanced settings toggle then. Where unchecking that box hides the buttons